apps/web/src/design-system/cards/Cards.tsx

```diff
@@ -3,22 +3,26 @@ import { useNavigate } from 'react-router-dom';
 import { Grid, Stack, useMantineColorScheme } from '@mantine/core';
 import styled from '@emotion/styled';
 
-import { When } from '../../../components/utils/When';
-import { colors, shadows, Text } from '../../../design-system';
+import { When } from '../../components/utils/When';
+import { colors, shadows } from '../config';
+import { Text } from '../typography/text/Text';
 
 interface ICardCell {
   navIcon?: (props: React.ComponentPropsWithoutRef<'svg'>) => JSX.Element | React.ReactNode;
   description?: React.ReactNode | string;
   navigateTo?: string;
+  onClick?: React.MouseEventHandler<HTMLButtonElement>;
   imagePath?: string;
   href?: string;
+  disabled?: boolean;
+  testId?: string;
 }
 
 export function Cards({ cells }: { cells: ICardCell[] }) {
```

Comment:
moved this component to the `design-system` folder and adjusted it a little to be more close with the designs